src/scala/org/pantsbuild/scoverage/report/ScoverageReport.scala

```diff
@@ -0,0 +1,254 @@
+package org.pantsbuild.scoverage.report
+
+import java.io.File
+import sbt.internal.util.ConsoleLogger
```

Comment:
This should probably not depend on SBT at all. Instead, should probably swap this to `slf4j-api` with `slf4j-jdk14` as the logging implementation. See https://www.slf4j.org/manual.html

Reply:
How about using the native Java logger (with a formatter). I'll define a simple one. Please review that in my next PR as well